src/formatter/builder.ts

```diff
@@ -99,28 +110,43 @@ const FormatterBuilder = {
     let Syntax = JavascriptSnippetSyntax
     if (doesHaveValue(snippetSyntax)) {
       const fullSyntaxPath = path.resolve(cwd, snippetSyntax)
-      Syntax = require(fullSyntaxPath) // eslint-disable-line @typescript-eslint/no-var-requires
+      Syntax = await importer(fullSyntaxPath, true)
```

Comment:
Occurs to me that we should add the same logic as was done for formatters, to differentiate between project-relative paths and module names - but that would technically be a breaking change? Maybe in a future PR

Reply:
Created by #1640